scripts/bundle.config.js

```diff
@@ -95,7 +95,7 @@ module.exports = (env = {}) => {
     // Set development mode (no minification)
     config.mode = 'development';
     // Remove .min from the name
-    config.output.filename = 'dist.js';
+    config.output.filename = 'dist/dist.js';
```

Comment:
The name is very nondescript. Call it `bundle.dev.js`?

Reply:
Off topic: we may want to inline source map with this build.

Reply:
> The name is very nondescript. Call it bundle.dev.js?

Yes I could agree that `dist` is a little non-descript, however wouldn't you expect the two modules to have the same name, one with `.min` and one without. That is how have been named in virtually every package I have used. The `dist` name was chosen to align with the existing name, which I have a sense that we do not want to change at this point...

> Off topic: we may want to inline source map with this build.

My thinking was that this would be an untranspiled build, so we could avoid the massive source map. Ideally we should use a bundler that generates a bundle with only our code, not the webpack scaffolding. I think rollup does this.